src/routes/about.route.js

```diff
@@ -0,0 +1,6 @@
+const express = require("express");
+const router = express.Router;
+
+router.get("/About", (req,res) => {
+    res.send("About Page");
```

Comment:
Remember to use status codes, also don't use capital letters in route names, this should be "/about"

Reply:
like `res.status(200).send("About Page);` ?

Reply:
Yup, that's exactly right :)